controls/pcidss_4_ocp4.yml

```diff
@@ -3023,11 +3051,20 @@ controls:
         - Internal systems receive time information only from designated central time server(s).
       levels:
         - base
-      status: pending
+      status: planned
       notes: |-
-        The selected rules might need updates in order to restrict their platform applicability
-        to avoid conflicts.
+        OpenShift is configured to use a public pool.ntp..org which is good and reliable, but
+        ultimatlely is not enterprise grade.
+        The notion of central time server implies that an exclusive NTP server is deployed to
+        serve the cluster.
+
+        NTP configuration is done at the node level, excpept when using Precison Time Procotol
+        (PTP), for which PTP Operator can be used. PTP can only be used on bare metal deployments.
+        https://docs.openshift.com/container-platform/latest/networking/ptp/about-ptp.html
       rules: []
+      related_rules:
```

Comment:
Not sure If below rules related. They are rhcos rules.
- service_chronyd_enabled
- service_chronyd_or_ntpd_enable

Reply:
I have added `service_chronyd_enabled`.
`service_chronyd_or_ntpd_enable` is very similar and`chronyd_specify_remote_server` was already there.

Reply:
previously, we don't have rhcos4 profile for pci dss: https://docs.openshift.com/container-platform/4.16/security/compliance_operator/co-scans/compliance-operator-supported-profiles.html#compliance-supported-profiles_compliance-operator-supported-profiles. Does it mean we will have rhcos4-pci-dss profile? Thanks.

Reply:
Potentially, but we've broken the work up into pieces with the profiles so we can deliver some value with the OpenShift profiles initially.